charts/datadog/README.md

```diff
@@ -617,6 +617,7 @@ helm install --name <RELEASE_NAME> \
 | datadog.podLabelsAsTags | object | `{}` | Provide a mapping of Kubernetes Labels to Datadog Tags |
 | datadog.processAgent.enabled | bool | `true` | Set this to true to enable live process monitoring agent |
 | datadog.processAgent.processCollection | bool | `false` | Set this to true to enable process collection in process monitoring agent |
+| datadog.processAgent.processDiscovery | bool | `false` | Enables autodiscovery of integrations |
```

Comment:
I think the CI is failing because the description here doesn't match the one in `charts/datadog/values.yaml `.
Running the script to regenerate the README should fix this.

Reply:
Ah I see. I ran `helm-docs.sh` and it fixed it